logpolicy/logpolicy.go

```diff
@@ -769,6 +769,11 @@ func NewLogtailTransport(host string, netMon *netmon.Monitor, logf logger.Logf)
 	}
 	tr.DialContext = MakeDialFunc(netMon, logf)
 
+	// We're uploading logs ideally infrequently, with specific timing that will
+	// change over time. Try to keep the connection open, to avoid repeatedly
+	// paying the cost of TLS setup.
+	tr.IdleConnTimeout = time.Hour
```

Comment:
Presumably I need to make the equivalent change on the server side, right?

If the server closes a idle connection after the default of 90s, then this is effectively the same as before?

Reply:
Yeah, there's more considerations server side which we'll discuss over there :D